tests/model_registry/conftest.py

```diff
@@ -188,16 +188,22 @@ def model_registry_instance_oauth_proxy(
 
 @pytest.fixture(scope="class")
 def model_registry_mysql_config(
-    model_registry_db_deployment: Deployment, model_registry_db_secret: Secret
+    model_registry_db_deployment: Deployment,
+    model_registry_db_secret: Secret,
+    ssl_ca: str | None = None,
+    port: int = 3306,
```

Comment:
I don't believe this is how a fixture works. It can request another fixture, but one does not pass args to it outside of request.param https://docs.pytest.org/en/stable/how-to/fixtures.html